consensus/src/quorum_store/tests/proof_manager_test.rs

```diff
@@ -60,24 +59,75 @@ async fn get_proposal_and_assert(
     );
     proof_manager.handle_proposal_request(req);
     let GetPayloadResponse::GetPayloadResponse(payload) = callback_rx.await.unwrap().unwrap();
-    if let Payload::InQuorumStore(proofs) = payload {
-        assert_eq!(proofs.proofs.len(), expected.len());
-        for proof in proofs.proofs {
-            assert!(expected.contains(&proof));
-        }
-    } else {
-        panic!("Unexpected variant")
+    payload
+}
+
+fn assert_payload_response(
+    payload: Payload,
+    expected: &[ProofOfStore],
+    max_txns_to_include_in_block: Option<usize>,
+) {
+    match payload {
+        Payload::InQuorumStore(proofs) => {
+            assert_eq!(proofs.proofs.len(), expected.len());
+            for proof in proofs.proofs {
+                assert!(expected.contains(&proof));
+            }
```

Comment:
do you want to assert_none!(max_txns_to_include_in_block) ?

Reply:
'max_txns_to_include_in_block' is from the test code, why burn some CPU cycles on that assert ? I would say lets not